avm/res/api-management/service/tests/e2e/developerSku/main.test.bicep

```diff
@@ -0,0 +1,54 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults with a Developer SKU'
```

Comment:
Do we want to show the usage of a developer sku or in particular a developer sku defaults?
- In the former case, please update title and description to not mention defaults.
- In the latter case, please remove any parameter which is not of type Required.